### DISCUSSION TITLE
Netbox Presentation Request

### DISCUSSION THREAD

**Discussion by mahmoudkhalil2** [netbox-community/netbox#12390] [Q&A]
Hello Team, I'm planing to present the Netbox tool to my team in my Company to show the Netbox features, benefits, capabilities and value.

So, I'm asking Does anyone have a power point or PDF presentation of the Netbox to use it during my meeting with my team.

Thanks in advance

**Comment by mahmoudkhalil2** [comment]
Dears, Does anyone can help?

**Comment by aopdal** [comment]
I'm always do a demo of NetBox and not a presentation you can use https://demo.netbox.dev/
